iocore/eventsystem/I_VConnection.h

```diff
@@ -116,13 +116,16 @@
 //
 //////////////////////////////////////////////////////////////////////////////
 
-/** Used in VConnection::shutdown(). */
-enum ShutdownHowTo_t {
-  IO_SHUTDOWN_READ = 0,
-  IO_SHUTDOWN_WRITE,
-  IO_SHUTDOWN_READWRITE,
+enum class IOShutdown {
+  READ      = SHUT_RD,
+  WRITE     = SHUT_WR,
+  READWRITE = SHUT_RDWR,
 };
 
+#define IO_SHUTDOWN_READ IOShutdown::READ
```

Comment:
`IO_SHUTDOWN_READ` is the name I actually wanted, and it's consistent with IOCore style and nomenclature. It's just not possible to alias it while using an enum class. Maybe I should just switch it back to a regular enum.